test/test_automl/test_estimators.py

```diff
@@ -465,6 +466,156 @@ def exclude(lst, s):
                 assert all(leaderboard['ensemble_weight'] > 0)
 
 
+@pytest.mark.parametrize('estimator', [AutoSklearnRegressor])
+@pytest.mark.parametrize('resampling_strategy', ['holdout'])
+@pytest.mark.parametrize('X', [
+    np.asarray([[1.0, 1.0, 1.0]] * 50 + [[2.0, 2.0, 2.0]] * 50)
```

Comment:
Maybe update `X` so that any regressor can make the mapping directly, 4 different inputs for 4 different outputs. Probably not a problem but it would make more sense.